Update SIGPLAN-AV page

janpaulpl opened the pull request.

sampsyo commented:
Hello, @janpaulpl! It would be really helpful if you could include some commentary about what changes this entails so I can navigate around them—any chance you could write some notes?

Also, it looks like this includes a Python script to generate HTML. Did you know that this entire site is already generated HTML, including from data files with personnel listings like this? It uses Jekyll, and the data comes from various YAML files. For example, here's the listing of the executive committee:
https://github.com/SIGPLAN/SIGPLAN.github.io/blob/9bd36de7f4050a1a5b79f1fa191abaff7180edcd/_data/Committees.yaml#L16-L28

And then it gets formatted into HTML using Liquid templates; namely:
https://github.com/SIGPLAN/SIGPLAN.github.io/blob/9bd36de7f4050a1a5b79f1fa191abaff7180edcd/ContactUs.md?plain=1#L11
https://github.com/SIGPLAN/SIGPLAN.github.io/blob/9bd36de7f4050a1a5b79f1fa191abaff7180edcd/_includes/committee-list.liquid

Do you think we could use that existing functionality to format the HTML you're looking for, instead of creating an *ad hoc* approach just for this one list?

janpaulpl commented:
No problem @sampsyo! We've removed all the ad-hoc HTML processing and replaced it with Liquid which is much cleaner. The PR also has a description of the changes now.

sampsyo commented:
Looks great; thanks!!
